hazelcast/src/main/java/com/hazelcast/internal/management/ManagementDataSerializerHook.java

```diff
@@ -41,7 +40,7 @@
     public static final int UPDATE_MANAGEMENT_CENTER_URL = 1;
     public static final int UPDATE_MAP_CONFIG = 2;
     public static final int MAP_CONFIG_DTO = 3;
-    public static final int ADD_WAN_CONFIG = 4;
+    //public static final int ADD_WAN_CONFIG = 4;
```

Comment:
maybe remove this line completely and shift other IDs up?

Reply:
Done, thanks for the tip!